engine/src/test/java/org/hibernate/search/test/metadata/Snafu.java

```diff
@@ -65,4 +65,15 @@
 	@Field
 	@org.hibernate.search.annotations.Analyzer(impl = FooAnalyzer.class)
 	private String custom;
+
+
+	// Just so that @Spatial works; see HSEARCH-2759
+	public double getLatitude() {
```

Comment:
Not really; the tests using this class do not attempt to index an instance, so previously the reflection code was never executed.

Now that we execute the reflection on startup, the bug (HSEARCH-2759) is more visible. It's always been here, though, it's just that we didn't really use the `@Spatial` feature (no indexing).